### DISCUSSION TITLE
Is there a way to import Remarks from one tab to another?

### DISCUSSION THREAD

**Discussion by Threewit** [rgriebl/brickstore#492] [Q&A]
Like the titles suggests, can i take the remarks from a store and add it to a set? or is that something i just have to do manually?

**Comment by ZZJHONS** [comment]
Edit -> Copy Values from Document

**Reply by Threewit** [reply]
I am doing that but nothing is happening do i merge or copy the remarks?

**Reply by ZZJHONS** [reply]
Try with the latest Dev Build that you can download here:

https://github.com/rgriebl/brickstore/actions/runs/1679332234

**Reply by Threewit** [reply]
Tested with the new build and i am still not getting anything. Is it supposed to be instant? sorry for being troublesome and thank you for your help.

**Reply by Threewit** [reply]
To clarify i am doing everything in this video just on windows and it is not doing anything. 

https://www.youtube.com/watch?v=XQG1YKbeKB8

Could it be a windows issue? or just bad luck on my part?
